A32NX/html_ui/Pages/VCockpit/Instruments/NavSystems/A320_Neo/A32NX_NavSystem.js

```diff
@@ -47,6 +47,12 @@ class NavSystem extends BaseInstrument {
             this.currFlightPlanManager.registerListener();
         }
         this.currFlightPlan = new FlightPlan(this);
+        if (typeof g_modDebugMgr != "undefined") {
+            g_modDebugMgr.AddConsole(null);
+        }
+        Include.addScript("/JS/debug.js", function () {
+            g_modDebugMgr.AddConsole(null);
+        });
```

Comment:
If we're gonna add this, we should probably have it do nothing if debug isn't installed.
```suggestion
        if (typeof g_modDebugMgr !== "undefined") {
            Include.addScript("/JS/debug.js", function () {
                g_modDebugMgr.AddConsole(null);
            });
        }
```

Reply:
oops, didn't mean to commit. but as you said might as well?